docs/index.md

```diff
@@ -1724,21 +1721,24 @@ Some shells support recursive matching by using the `**` wildcard in a glob. Bas
 $ mocha "./spec/**/*.js"
 ```
 
-*Note*: Double quotes around the glob are recommended for portability.
+_Note_: Double quotes around the glob are recommended for portability.
```

Comment:
**Wrong!** For portability, single quotes should be used, not double...

Reply:
no, that doesn't work on windows.  cmd or powershell, I forget.

Reply:
We should find out then. Leaving this up to each individual shell (given the varying levels of glob support) is _definitely_ not portable.

For UNIX, single quotes will be the most portable since they will prevent the shell from doing the expansion. For _maximum_ portability and reproducability, Mocha **must** handle the globs itself.

Reply:
Medium article "[Why you should always quote your globs in NPM scripts](https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784)".